packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -438,7 +452,7 @@ class AndroidStudio implements Comparable<AndroidStudio> {
     return keyMatcher.stringMatch(plistValue)?.split('=').last.trim().replaceAll('"', '');
   }
 
-  void _init({Version? version}) {
+  void _init() {
```

Comment:
can this be moved into the constructor?

Reply:
It could be; but, given how heavy the initialization logic is, I decided to instead just rename the method with a slightly more descriptive name.

There's definitely potential here to refactor the class to not have a half-initialized state (pre-constructor time), but it would result in a pretty big diff. I think I'll leave that out of scope for this change.